packages/ember-runtime/lib/mixins/mutable_array.js

```diff
@@ -388,7 +388,7 @@ export default Mixin.create(EmberArray, MutableEnumerable, {
     @public
   */
   addObject(obj) {
-    if (!this.contains(obj)) {
+    if (!this.includes(obj)) {
```

Comment:
Sure. Moreover, I think I have to use includes instead of contains only behind feature flag, right?